tests/parametric/test_span_sampling.py

```diff
@@ -753,3 +753,78 @@ def test_entire_trace_dropped_when_dropping_policy_is_active018(self, test_agent
         traces = test_agent.wait_for_num_traces(0, clear=True)
 
         assert len(traces) == 0
+
+
+    @bug(context.library == "cpp", reason="APMAPI-1545")
```

Comment:
Created a ticket for this issue, hopefully at the right place with the right fields
https://datadoghq.atlassian.net/browse/APMAPI-1545